test/framework/integration_framework/test_irohad.hpp

```diff
@@ -38,6 +38,7 @@ namespace integration_framework {
                bool is_mst_supported)
         : Irohad(block_store_dir,
                  pg_conn,
+                 "127.0.0.1",
```

Comment:
It might be useful to pass this from IntegrationTestFramework, similar to Torii port.